src/index.js

```diff
@@ -1,14 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import { BrowserRouter as Router } from 'react-router-dom';
+// import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './redux/configureStore';
 import './index.css';
 import App from './App';
 
+// const root = ReactDOM.createRoot(document.getElementById('root'));
+// root.render(
+//   <React.StrictMode>
+//     <Router>
+//       <App />
+//     </Router>
+//   </React.StrictMode>,
+// );
+
```

Comment:
- [ ] [OPTIONAL] Kindly remove the unused lines of code in order to have a clean code base.